checks/mixins/sarus_extra_launcher_options.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2016-2023 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+
+
+class SarusExtraLauncherOptionsMixin(rfm.RegressionMixin):
+    @run_before('run')
+    def set_launcher_options(self):
+        if self.current_system.name in {'hohgant'}:
```

Comment:
Not great to have `hohgant` hardcoded here + the flag  does not seem to be used: see rfm_job.sh of check_gpu_ids.py for example

Reply:
On check_gpu_ids, it's not used because it's not an mpi job.